### DISCUSSION TITLE
what the learning curve for julia like

### DISCUSSION THREAD

**Discussion by MAJIDNISAR** [github/india#76] [Open Source] [answered]
knowing most of the modern day programing languages, what is it like to learn julia , time frame...etc

**Comment by ViralBShah** [comment]
If you are familiar with say,  python, I think you can be up and running in Julia in a matter of a few days. Mastering multiple dispatch, Julia's key paradigm can take bit longer - but people often get their first non-trivial Julia programs going in a couple of days.

**Reply by MAJIDNISAR** [reply]
Thankyou, for the quick reply, will be taking a deeper dive, on a side note any vs code extensions you suggest for setup ...

**Reply by ViralBShah** [reply]
Yes - use https://www.julia-vscode.org/

**Comment by ViralBShah** [comment, marked as answer]
See https://julialang.org/learning/ for many different learning resources.
